.github/workflows/tics.yml

```diff
@@ -86,3 +86,15 @@ jobs:
         viewerUrl: https://canonical.tiobe.com/tiobeweb/TICS/api/cfg?name=default
         ticsAuthToken: ${{ secrets.TICSAUTHTOKEN }}
         installTics: true
+
+    - name: Report workflow failure
+      if: ${{ failure() }}
```

Comment:
Should this include `cancelled()`? That would at least help with confirming this works, after merging.

Reply:
Should the message be reworded then? Something like `Scheduled TICS workflow exited before completing`?